We ship pycryptodome as full replacement of pycrypto, so the dependency is on
pycryptodome, not pycryptodomex.
diff --git a/requirements/base.txt b/requirements/base.txt
index 27923c0..088dbb0 100644
--- a/requirements/base.txt
+++ b/requirements/base.txt
@@ -1,5 +1,5 @@
 Jinja2
-msgpack>=0.5,!=0.5.5,<1.0.0
+msgpack>=0.5,!=0.5.5,<1.1.0
 PyYAML
 MarkupSafe
 requests>=1.0.0
diff --git a/requirements/crypto.txt b/requirements/crypto.txt
index 00e98d3..781a89f 100644
--- a/requirements/crypto.txt
+++ b/requirements/crypto.txt
@@ -1 +1 @@
-pycryptodomex>=3.9.7
+pycryptodome>=3.9.7
